charmcraft/logsetup.py

```diff
@@ -25,8 +25,11 @@
 FORMATTER_SIMPLE = "%(message)s"
 FORMATTER_DETAILED = "%(asctime)s  %(name)-30s %(levelname)-8s %(message)s"
 
-_logger = logging.getLogger("charmcraft")
-_logger.setLevel(logging.DEBUG)
+logger = logging.getLogger("charmcraft")
+enabled_loggers = [
+    logging.getLogger("charmcraft"),
```

Comment:
Maybe just use `logger` and not re-ask it?

Reply:
As far as the names, I use __name__ with getLogger so it's not trivial to shorten it.  I can throw another tab or two in there to improve alignment though.

Reply:
Fixed

- Added 10 more spaces which should be plenty.
- Re-used `logger` too.